src/main/java/org/jfrog/hudson/ArtifactoryBuilder.java

```diff
@@ -40,12 +45,13 @@
 import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.interceptor.RequirePOST;
 
-import javax.servlet.ServletException;
 import java.io.IOException;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
+import static org.jfrog.hudson.util.ProxyUtils.createProxyConfiguration;
+
 /**
  * @author Yossi Shaul
  */
```

Comment:
I don't think it's possible because this is a part of the path of the global configuration file:
`$JENKINS_HOME/org.jfrog.hudson.ArtifactoryBuilder.xml`
Changing ArtifactoryBuilder's name may cause serious problems.